modules/ROOT/pages/release-notes.adoc

```diff
@@ -23,6 +23,10 @@ These credentials can be consulted and modified:
 * in the <BONITA>/server/conf/tomcat-users.xml file in a *Bonita Tomcat bundle*
 * using xref:bonita-docker-installation.adoc#_monitoring_username[specific Environment parameters] when starting a *Bonita Docker image*
 
+=== Other changes
+
+* When the engine fails to evaluate output operations of a connector, the connector is no more automatically retried. It could be an issue to re-execute a connector because, connectors might have side effects. It is the choice of the administrator of the platform to re-execute the connector or not. See xref:runtime:connectors-execution.adoc[documentation about connector execution] for details about connector failure and replay.
```

Comment:
```suggestion
* When the engine fails to evaluate output operations of a connector, the connector is no more automatically retried. It could be an issue to re-execute a connector because connectors might have side effects. It is the choice of the administrator of the platform to re-execute the connector or not. See xref:runtime:connectors-execution.adoc[documentation about connector execution] for details about connector failure and replay.
```